Sources/KSCrashRecording/Monitors/KSCrashMonitor_AppState.c

```diff
@@ -460,6 +458,11 @@ const KSCrash_AppState* const kscrashstate_currentState(void)
     return &g_state;
 }
 
+static const char* name(void)
+{
+    return "KSCrashMonitorTypeApplicationState";
```

Comment:
Talking performance, having prefix like `KSCrashMonitorType` in these methods is not great as it's going to be checked every time. Not a big deal, but `ApplicationState_KSCrashMonitorType` or simply `ApplicationState` would work just a bit faster.